bundles/create_openhab_binding_skeleton.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Is this change a mistake? are you able to remove it from showing up and being tracked?

Reply:
This abstracts over the concrete install path of the bash binary. It's needed for distros using an advanced package manager (nix) and it still works in traditional envs. You should really consider keeping it.

Reply:
Over my head. I feel it is best for a separate PR which would be great to have. A single line change like that will progress much quicker.

Reply:
@Skinah Sure, I can make a separate PR for that